symfit/core/minimizers.py

```diff
@@ -129,6 +130,38 @@ def resized(*args, **kwargs):
         return resized
 
 
+class HessianMinimizer(GradientMinimizer):
+    @keywordonly(hessian=None)
+    def __init__(self, *args, **kwargs):
+        hessian = kwargs.pop('hessian')
+        super(HessianMinimizer, self).__init__(*args, **kwargs)
+        if hessian is not None:
+            hess_with_fixed_parameters = partial(hessian, **{p.name: p.value for p in self._fixed_params})
+            self.wrapped_hessian = self.resize_hess(hess_with_fixed_parameters)
+        else:
+            self.wrapped_hessian = None
+
+    def resize_hess(self, func):
```

Comment:
I think this code is identical to resize_jac, correct? Since we inherit from `GradientMinimizer`, why not just call resize_jac? After the resize has happened nobody will ever know anyway, since the new function will mascarade as `resized`.